pypeit/core/telluric.py

```diff
@@ -321,7 +379,87 @@ def shift_telluric(tell_model, loglam, dloglam, shift, stretch):
     return tell_model_shift
 
 
-def eval_telluric(theta_tell, tell_dict, ind_lower=None, ind_upper=None):
+def eval_telluric_pca(theta_tell, tell_dict, ind_lower=None, ind_upper=None):
+    """
+    Evaluate the telluric PCA model.
+
+    The parameters provided by ``theta_tell`` must be: ntell PCA coefficients,
+    spectral resolution, shift, and stretch. The latter two can be omitted if
+    a shift and stretch of the telluric model are not included.
+
+    This routine performs the following steps:
+
+       1. summation of telluric PCA components multiplied by coefficients
+       
+       2. transformation of telluric PCA model from arsinh(tau) to transmission
+
+       3. convolution of the atmosphere model to the resolution set by
+          the spectral resolution.
+
+       4. (Optional) shift and stretch the telluric model.
+
+    Args:
+        theta_tell (`numpy.ndarray`_):
+            Vector with the telluric PCA coefficients
+        tell_dict (:obj:`dict`):
```

Comment:
Can you specify which parameters fill which indices in the theta_tell in the docs please.

Reply:
I added a bit here but was somewhat more explicit in the docs to `tellfit`.